source/administration/devops-command-center.rst

```diff
@@ -3,199 +3,255 @@ Mattermost Incident Management
+* Run the ``/incident restart`` slash command from within the incident channel.
+* From the desktop or browser:
+  1. Select the **Incidents** icon in the channel header to open the RHS.
+  2. Browse to the **Tasks** tab.
+  3. Click **Restart Incident**.
 
-Incident status and information
--------------------------------
+Reviewing past incidents
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-To view the status of your active incident(s) select **Main Menu > Playbooks & Incidents**. Select the **Incident** tab to view a list of incidents in your team. Select the incident name to view a summary of the incident, jump to the channel, or export the channel.
+To view past incidents of which you are a member, from the desktop or browser:
+1. Navigate to **Main Menu > Playbooks & Incidents**.
+2. Select the **Incidents** tab.
+3. Find the incident to be reviewed and click the name of the incident.
+4. Review the incident details:
+   * The duration, total number of members ever involved, and messages sent in the channel are listed.
+   * A graphical timeline shows how much time elapsed between completing tasks.
+5. Optionally export the contents of the incident channel for review during a post-mortem.
+
+Built-in slash commands
+-----------------------
 
-To view information about ongoing incidents, select the **Incidents** icon in the header of any channel to open the RHS where all ongoing incidents are listed. Select **Go to Incident Channel** to open the relevant channel and see:
+The ``/incident`` slash command allows interacting with incidents via the post textbox on desktop, browser, and mobile:
 
-- The incident commander
-- The current stage
-- The remaining tasks
-- The finished tasks
+- ``/incident start`` - Start a new incident.
+- ``/incident end`` - End an ongoing incident.
+- ``/incident restart`` - Restart an ended incident.
+- ``/incident check [stage #] [task #]`` - Check/uncheck the specified stage's task.
+- ``/incident announce ~channel`` - Announce the current incident in other channels.
+- ``/incident list`` - List all your incidents.
+- ``/incident commander [@username]`` - Show or change the current commander.
+- ``/incident info`` - Show a summary of the current incident.
+- ``/incident stage [next/prev]`` - Move to the next or previous stage.
 
-You can also:
+Generating test data
+^^^^^^^^^^^^^^^^^^^^
 
-- Assign a step to yourself or another incident member.
-- Mark a step as **Complete** or **Incomplete**.
-- Start an automated action.
-- Invite new members to the channel.
+To quickly Mattermost Incident Management, use the following test commands to create incidents populated with random data:
+- ``/incident test create-incident [playbook ID] [timestamp] [incident name]`` - Provide the ID of an existing playbook to which the current user has access, a timestamp, and an incident name. The command creates an ongoing incident with the creation date set to the specified timestamp.
```

Comment:
If there's a specific timestamp format required, consider specifying that format.

Reply:
@agarciamontoro, is there a specific format required?